### PR TITLE
add: fontSize prop for IconStore

### PR DESCRIPTION
**Pull request by Gauravsharma-20**
add fontSize prop in IconStore for it to work with Typography changes.

### CODE REVIEW

File: src/components/atoms/IconStore/IconStore.tsx [hunk 1]
```diff
@@ -1,12 +1,17 @@
-import React from 'react';
+import "./iconStore.css";
```
Comment (StTronn) [inline src/components/atoms/IconStore/IconStore.tsx:1]: css should be at bottom

File: src/components/atoms/IconStore/IconStore.tsx [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
-import { IMG_ICON_LIST, MI_ICON_LIST } from './iconList';
-import { Image } from '../Image';
+import React from "react";
 
-import { Theme } from '../../../utils/types/theme';
-import { listenToCustomEvent, unListenToCustomEvent } from '../../../utils/helper';
-
-import './iconStore.css';
+import {
+  listenToCustomEvent,
```
Comment (StTronn) [inline src/components/atoms/IconStore/IconStore.tsx:6]: fix spacing

File: src/components/atoms/IconStore/IconStore.tsx [hunk 3]
```diff
@@ -57,15 +62,23 @@ class IconStore extends React.PureComponent<Props, State> {
 
     } else {
       const MI_NAME = MI_ICON_LIST[iconName as keyof typeof MI_ICON_LIST];
+      let fontSizeStyle = width;
+
+      if (fontSize) {
+        fontSizeStyle = fontSize;
+      }
+
```
Comment (StTronn) [inline src/components/atoms/IconStore/IconStore.tsx:70]: can we use the ternary operator instead ?

File: src/components/atoms/IconStore/IconStore.tsx (hunk 1 above)
Comment (Gauravsharma-20) [reply src/components/atoms/IconStore/IconStore.tsx]: reverted

File: src/components/atoms/IconStore/IconStore.tsx (hunk 3 above)
Comment (Gauravsharma-20) [reply src/components/atoms/IconStore/IconStore.tsx]: Done

**Review by StTronn — DISMISSED** [review]

**Review by StTronn — APPROVED** [review]

**Pull request merged**